students/WWhite/lesson07/html_render.py

```diff
@@ -0,0 +1,105 @@
+/>:class Element:
+    indentation = "  "
+    tag_class = ""
+    attributes = ""
```

Comment:
`attributes` should be an instance attribute since the value can be different for different subclass instances